src/mousedownobserver.js

```diff
@@ -0,0 +1,40 @@
+/**
```

Comment:
This could end in the engine as `MouseObserver` (which supports mousedown, move and up).